db/schema.rb

```diff
@@ -840,7 +839,8 @@
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
+    t.string "{:null=>false}"
```

Comment:
This seems incorrect? Some of these changes seem unrelated to the code here.

Reply:
@dorner Sir, i check it out schema but i didnt found this line now,